packages/nns/src/canisters/governance/response.converters.ts

```diff
@@ -382,6 +382,8 @@ const toAction = (action: RawAction): Action => {
             snsTokenE8s: params.sns_token_e8s,
             maxParticipantIcpE8s: params.max_participant_icp_e8s,
             minIcpE8s: params.min_icp_e8s,
+            neuronBasketConstructionParameters:
+              params.neuron_basket_construction_parameters[0],
```

Comment:
you can maybe use `toNullable` or `fromNullable` here?

Reply:
I always forget these helpers 😅